proposals/0242-alert-style-subtle.md

```diff
@@ -6,125 +6,203 @@
+    </element>
+</enum>
 
-    <param name="alertStylesAvailable" type="AlertStyle" array="true" minValue="1" maxValue="1000" mandatory="false" since="6.x">
-        <description>An array of available alert styles. If this parameter is not provided, only the `PROMINENT` style is assumed to be available.</description>
+<enum name="ImageFieldName" since="1.0">
+    <!-- New Values -->
+    <element name="subtleAlertIcon">
+        <description>The image of the subtle alert; applies to `SubtleAlert` `alertImage`</description>
+    </element>
+    <element name="subtleAlertSoftButtonImage">
+        <description>An image field in the soft button of a subtle alert; applies to `SubtleAlert` `softButtons`</description>
+    </element>
+</enum>
+```
+
+#### Subtle Alert
+```xml
+<function name="SubtleAlert" functionID="SubtleAlertID" messagetype="request">
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
+        
+     <param name="alertText1" type="String" maxlength="500" mandatory="false">
+         <description>The first line of the alert text field</description>
+     </param>
+     
+     <param name="alertText2" type="String" maxlength="500" mandatory="false">
+         <description>The second line of the alert text field</description>
+     </param>
+
+    <param name="alertIcon" type="Image" mandatory="false">
+        <description>
+            Image to be displayed for the corresponding alert. See Image. 
+            If omitted, no (or the default if applicable) icon should be displayed.
+        </description>
     </param>
-</struct>
+     
+     <param name="ttsChunks" type="TTSChunk" minsize="1" maxsize="100" array="true" mandatory="false">
+         <description>
+            An array of text chunks of type TTSChunk. See TTSChunk.
+            The array must have at least one item.
+         </description>
+     </param>
+     
+     <param name="duration" type="Integer" minvalue="3000" maxvalue="10000" defvalue="5000" mandatory="false">
+         <description>
+            Timeout in milliseconds.
+            Typical timeouts are 3-5 seconds.
+            If omitted, timeout is set to 5s.
+         </description>
+     </param>
+     
+     <param name="softButtons" type="SoftButton" minsize="0" maxsize="2" array="true" mandatory="false" since="2.0">
+        <description>
+            App defined SoftButtons.
+            If omitted on supported displays, the displayed alert shall not have any SoftButtons.
+        </description>
+     </param>
+
+     <param name="cancelID" type="Integer" mandatory="false">
+        <description>
+            An ID for this specific alert to allow cancellation through the `CancelInteraction` RPC.
+        </description>
+    </param>
+</function>
+
+<function name="OnSubtleAlertPressed" functionID="OnSubtleAlertPressedID" messagetype="notification">
```

Comment:
I think this needs to be added to the HMI API as well.

Reply:
👌🏻